### ISSUE TITLE
Node modules missing when running 'tns platform add android'

### ISSUE THREAD

**Issue by N3ll** [NativeScript/nativescript-cli#1181]
_From @arlowhite on November 12, 2015 21:32_

I installed node with nvm
Node v4.2.1
npm 3.3.12

Installed NativeScript with:
`npm install nativescript -g`

When I tried 'tns platform add android', I got errors for missing modules, starting with:
> Cannot find module 'child-process-close'

I had to install all of these modules for the command to finally work.
```
npm install -g child-process-close
npm install -g config-chain
npm install -g nopt
npm install -g umask
npm install -g npmlog
npm install -g uid-number
npm install -g abbrev
npm install -g npm-registry-client
npm install -g chownr
npm install -g npm-cache-filename
npm install -g write-file-atomic
npm install -g char-spinner
npm install -g read-package-json
npm install -g npm-package-arg
```

Does NativeScript need to list these as dependencies?

_Copied from original issue: NativeScript/NativeScript#1095_

**Comment by rosen-vladimirov** [comment]
Hi @arlowhite, 
I'm not sure what happened, but `nativescript-cli` does not depend on any of the packages that you've installed. Also in order to work correctly, `nativescript-cli` **does not** require global installation of any other package (`-g` flag). Do you have these dependencies listed in the package.json of your project for example?

**Comment by arlowhite** [comment]
**Fixed by reinstalling nativescript**

tns version is 1.4.3

I'm going through the getting started guide.
http://docs.nativescript.org/getting-started
Followed the instructions in **1.1 Start your app** (clone sample-Groceries, checkout start)

I removed the first module I had problems with to reproduce the error.
`npm uninstall -g child-process-close`

I get this error with the create command as well:
> awhite@ArloDesktop:~/Code/ng2/NativeScript$ tns create BlankApp
> Cannot find module 'child-process-close'

**FIXED**
I just removed all the modules I installed globally and did a uninstall/install of nativescript, and now it works. Maybe there was an npm bug? I think I upgraded it since the first time I installed nativescript. Otherwise maybe my first install had an error I missed.

Sorry for the false-positive.
